packages/connect-wallet-modal/src/connectButtons/ConnectInjected.tsx

```diff
@@ -42,12 +44,15 @@ export const ConnectInjected: FC<ConnectInjectedProps> = (
     if (await detector?.()) {
       disconnect?.();
       await connectAsync({ connector });
+    } else if (isMobileOrTablet && deeplink) {
+      openWindow(deeplink);
     } else if (downloadURLs) {
       suggestApp(downloadURLs);
     }
   }, [
     connectAsync,
     connector,
+    deeplink,
```

Comment:
shouldn't isMobileOrTablet be a dependency too?

Reply:
@DiRaiks no, because `isMobileOrTablet` is non-reactive and is set only once on the initial page load. Also checked that eslint is not complaining.